docs/UserGuide.md

```diff
@@ -130,8 +129,10 @@ Displays all grading components and their respective weights for a class.
     - `list_component`
 
 ---
-
-
+## Notes:
+- Matric Number (`i/` argument) is case insensitive. Therefore, `A1234567X` is the same as `a1234567x`. Matric numbers 
+will be converted to uppercase for storage. 
+- (coming soon) All other arguments are case insensitive and will be converted to lowercase for storage. 
```

Comment:
Will names be stored as all lowercase a well? If we do want to store as all lower or upper case, perhaps it could be more "formal" to store names as all uppercase?

Reply:
{Repeated message from above}

Just leaving this here, this is our agreement from our meeting:

Display and Storage:

    matricNumber --> convert to UPPERCASE
    Other data --> keep as original case

Matching and Finding:

    Set to UPPERCASE before comparison

Will open an issue for it, issue https://github.com/AY2425S1-CS2113-W13-4/tp/issues/102